tools/batch/batch_states/compile_notebook.py

```diff
@@ -0,0 +1,100 @@
+import os
+import argparse
+import sys
+import re
+
+HOME_PATH = os.path.expandvars('$HOME') + "/work/gluon-nlp"
```

Comment:
use [`os.path.join`](https://docs.python.org/3/library/os.path.html#os.path.join) for portability.